File: SecondoAnno/Java/ESAMI/Esame19giugno2017/SuperStore.java
public class SuperStore extends Negozio {

    private String ragioneSociale;
    private int partitaIva, nCasse;
    
    public SuperStore(String tipo, int codice, String indirizzo, int superficie, String ragioneSociale, int partitaIva, int nCasse) {
        super(tipo, codice, indirizzo, superficie);
        this.ragioneSociale = ragioneSociale;
        this.partitaIva = partitaIva;
        this.nCasse = nCasse;
    }

    public String getTipo() {
        return tipo;
    }

    public String getRagione() {
        return ragioneSociale;
    }
    
    public String toString() {
        return "Super-Store" + "\t" + codice + "\t" + indirizzo + "\t" + superficie + "\t" + "-" + "\t" + "-" + "\t" + ragioneSociale + "\t" + partitaIva + "\t" + nCasse; 
    }

}

File: SecondoAnno/Programmazione_II/Java/ESAMI/Esame19giugno2017/Python/super_store.py
from negozio import Negozio

class SuperStore:

    ragione_sociale: str
    partita_iva: int
    n_casse:int 

    def __init__(self, tipo: str, codice: int, indirizzo: str, superficie: int, ragione_sociale: str, partita_iva: int, n_casse: int) -> None:
        super.__init__(tipo, codice, indirizzo, superficie)
        self.ragione_sociale = ragione_sociale
        self.partita_iva = partita_iva
        self.n_casse = n_casse

    def __str__(self) -> str:
        return "Super-Store" + "\t" + super().__str__()+ "\t-\t-\t" + self.ragione_sociale + "\t" + str(self.partita_iva) + "\t" + self.n_casse